#include <iostream>

void exception_example();